### DISCUSSION TITLE
Wrong link in the readme.me

### DISCUSSION THREAD

**Discussion by Kamikaze01git** [007revad/Synology_DSM_reinstall#19] [General]
First of all: thank u for your great work.
I really like it and it is very useful for my usecases!!
Thank you, thank you, thank you !!

I think there is a wrong link in the [readme.md](https://github.com/007revad/Synology_DSM_reinstall/blob/main/README.md)
The download link refferes to the zip of another from your great scripts ;-)

Cheers 😎🤜🤛

**Comment by 007revad** [comment]
Thanks.

 I just checked the readme of all my other scripts. Thankfully this was the only one with the wrong link that needed fixing.
